kafka_consumer/datadog_checks/kafka_consumer/data/conf.yaml.example

```diff
@@ -90,14 +90,34 @@ instances:
     # monitor_all_broker_highwatermarks: false
 
     ## @param tags - list of strings - optional
-    ## List of tags to attach to every metric and service check emitted by this integration.
+    ## A list of tags to attach to every metric and service check emitted by this instance.
     ##
     ## Learn more about tagging at https://docs.datadoghq.com/tagging
     #
     # tags:
     #   - <KEY_1>:<VALUE_1>
     #   - <KEY_2>:<VALUE_2>
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Overrides any `service` defined in the `init_config` section.
+    #
+    # service: <SERVICE>
```

Comment:
can you bump datadog-checks-base to 11.0.0?

Reply:
It's already required https://github.com/DataDog/integrations-core/blob/9cf3a3a7a85017956c9d9af6e897f33a4a8b00dc/kafka_consumer/setup.py#L31